devops/Dockerfile.worker

```diff
@@ -31,13 +31,19 @@ RUN curl -sSL https://install.python-poetry.org | python3 -
 RUN poetry config virtualenvs.create false
 COPY devops/pyproject.toml devops/poetry.lock ./
 
-RUN poetry install
+# use local mapillary (instead of directly from github in pyproject.toml)
+# due to issues with worker image environment
+RUN git clone https://github.com/mapillary/mapillary_tools.git /opt/mapillary_source
+RUN cd /opt/mapillary_source && git checkout 4bfebf8
```

Comment:
John: can we centralize this 4bfebf8

Reply:
@taoteg , fixed as could revert to adding papillary github and commit into pyproject.toml without any issues. I think now the lock file is fixed so I am not getting any more issues recently.